package.json

```diff
@@ -5,51 +5,48 @@
   "main": "index.html",
   "dependencies": {
     "@fortawesome/fontawesome-free": "^6.2.0",
+    "@mapbox/geojson-extent": "^1.0.1",
```

Comment:
Interesting. Is there a difference between this and [@turf/bbox](https://www.npmjs.com/package/@turf/bbox)?

Reply:
No idea, opened an issue to investigate this #773